### ISSUE TITLE
Ruby 3.0 uninitialized constant Gem::Release::Files::Template::FileUtils

### ISSUE THREAD

**Issue by CAMOBAP** [svenfuchs/gem-release#100]
```
ERROR:  While executing gem ... (NameError)
    uninitialized constant Gem::Release::Files::Template::FileUtils
Did you mean?  FileTest
	/Users/camobap/.rbenv/versions/3.0.1/lib/ruby/gems/3.0.0/gems/gem-release-2.2.1/lib/gem/release/files/template.rb:33:in `write'
	/Users/camobap/.rbenv/versions/3.0.1/lib/ruby/gems/3.0.0/gems/gem-release-2.2.1/lib/gem/release/cmds/gemspec.rb:71:in `generate'
	/Users/camobap/.rbenv/versions/3.0.1/lib/ruby/gems/3.0.0/gems/gem-release-2.2.1/lib/gem/release/cmds/gemspec.rb:63:in `block in run'
	/Users/camobap/.rbenv/versions/3.0.1/lib/ruby/gems/3.0.0/gems/gem-release-2.2.1/lib/gem/release/cmds/base.rb:88:in `block in in_dirs'
	/Users/camobap/.rbenv/versions/3.0.1/lib/ruby/gems/3.0.0/gems/gem-release-2.2.1/lib/gem/release/context/paths.rb:25:in `block (2 levels) in in_dirs'
	/Users/camobap/.rbenv/versions/3.0.1/lib/ruby/gems/3.0.0/gems/gem-release-2.2.1/lib/gem/release/context/paths.rb:9:in `chdir'
	/Users/camobap/.rbenv/versions/3.0.1/lib/ruby/gems/3.0.0/gems/gem-release-2.2.1/lib/gem/release/context/paths.rb:9:in `in_dir'
	/Users/camobap/.rbenv/versions/3.0.1/lib/ruby/gems/3.0.0/gems/gem-release-2.2.1/lib/gem/release/context/paths.rb:25:in `block in in_dirs'
	/Users/camobap/.rbenv/versions/3.0.1/lib/ruby/gems/3.0.0/gems/gem-release-2.2.1/lib/gem/release/context/paths.rb:23:in `each'
	/Users/camobap/.rbenv/versions/3.0.1/lib/ruby/gems/3.0.0/gems/gem-release-2.2.1/lib/gem/release/context/paths.rb:23:in `in_dirs'
	/Users/camobap/.rbenv/versions/3.0.1/lib/ruby/gems/3.0.0/gems/gem-release-2.2.1/lib/gem/release/context.rb:31:in `in_dirs'
	/Users/camobap/.rbenv/versions/3.0.1/lib/ruby/gems/3.0.0/gems/gem-release-2.2.1/lib/gem/release/cmds/base.rb:86:in `in_dirs'
	/Users/camobap/.rbenv/versions/3.0.1/lib/ruby/gems/3.0.0/gems/gem-release-2.2.1/lib/gem/release/cmds/gemspec.rb:61:in `run'
	/Users/camobap/.rbenv/versions/3.0.1/lib/ruby/gems/3.0.0/gems/gem-release-2.2.1/lib/gem/release/cmds/runner.rb:19:in `run_cmd'
	/Users/camobap/.rbenv/versions/3.0.1/lib/ruby/gems/3.0.0/gems/gem-release-2.2.1/lib/gem/release/cmds/runner.rb:8:in `run'
	/Users/camobap/.rbenv/versions/3.0.1/lib/ruby/gems/3.0.0/gems/gem-release-2.2.1/lib/gem/release/support/gem_command.rb:54:in `execute'
```

AFAIK simple `require "fileutils"` will resolve this issue

**Comment by PikachuEXE** [comment]
I guess this was never caught in spec since `fileutils` is required in a spec support file.
Fix released in `2.2.2`

**Comment by CAMOBAP** [comment]
@PikachuEXE thanks a lot, I can confirm the fix
